db/changelog/001-create-timeEntry-and-timePeriodType-table.json

```
@@ -0,0 +1,18 @@
+{
+  "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "002-create-timeEntry-and-timePeriodType-table",
```

Ah whoops! Missed the ID here
```suggestion
        "id": "001-create-timeEntry-and-timePeriodType-table",
```